examples/test_error.py

```diff
@@ -150,33 +147,32 @@
         signal_pathway,
         error_pathway,
         indirect="nScans",
-        fl=fl,
         return_frq_slice=True,
     )
     averaged_inactive_error = averaged_inactive.get_error()
     avg_avg_error = averaged_inactive_error.mean().item()
     # }}}
 
     # {{{ Calculating propagated error along active CT on noise slice
-    active_error = active_propagation(s, signal_pathway, indirect="nScans", fl=fl)
+    active_error = active_propagation(s, signal_pathway, indirect="nScans")
     avg_active_error = active_error.C.mean("nScans").item()
     # }}}
 
     # {{{ Plotting Errors
-    fl.next("comparison of std")
+    fl.next("comparison of std", legend=True)
     for i in range(len(s_int_lst)):
```

Comment:
this positions the legend to the right of the figure, and the \n below make sure everything fits in the legend box